data/quarg/quarg missions.txt

```diff
@@ -325,3 +325,150 @@ mission "Quarg Pug Arfecta Warning"
+			`You've joined the Lunarium in Coalition space, and even served as a messenger to bring the Quarg some information about the Lunarium itself, in the hopes that they might provide some help. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Lunarium and from your visit to the Quarg with Oobat?`
+			choice
+				`	(Yes.)`
+					goto yes
+				`	(Not now.)`
+					defer
+				`	(No, I'm not interested.)`
+					decline
+			label yes
+			`	You're never quite sure who to speak with when it comes to the Quarg, but you're instinctively drawn to one that stands out. You judge it to be an older member of their species; its thin scales cover its fingers, and the more bony parts of its head have turned to a bright, silvery shade. Unlike most Quarg you've seen, it is wearing a pale, faded white robe, with patches of red-colored chains (that somehow perfectly transition into fabric again) becoming more and more prominent closer to the top, culminating in a firm, crimson hood covering its head. Most surprising of all for a Quarg, however, is the large scar taking up much of the left side of its face, completely covering where its left eye should be.`
+			`	It is completely static, with a lost gaze focused on the ceiling here until you approach it. It doesn't squat down to meet your eye level. Instead it says nothing, simply gazing at you with its one eye, colored a bright gold.`
+			choice
+				`	"Hello, I'm <first> <last>. What can you tell me about the Coalition?"`
+				`	"Hello. Is it true that the Heliarchs defeated you and stole your ringworlds?"`
+			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down it finally speaks up.`
+			`	"Those three species arose in relative rapid succession, all in close proximity, compared to how other species tend to be separated by long periods of time, and longer still their distances. So large are the gaps, that one might never learn of those that came before. But, that was not the case for the three. The first of them, the Saryds, were in many ways much like us Quarg in our younger days, and we perhaps were much too lenient because of that... The Kimek, traumatized by the billions fallen in their famines, sought help to make sure they would never face such a crisis again, and were eager to establish friendly relations with the Saryds. The Arachi, in a way, parallel humanity's development, though they were less keen on letting their inner conflicts be made public. They were the last of the three, and upon meeting their two seniors, invested heavily in research to catch up.`
+			`	"In the intersecting point of their territories, we built our ringworlds. Seven thousand Earth years ago, the three species of the Coalition sought to take over those rings. After centuries... no, a millennium of fighting, we left that part of the galaxy."`
```

Comment:
We've described this Quarg in great detail, then have this action where the Quarg decides to only speak to you in a specific section of the spaceport, but then we immediately jump into lore dumping. I feel like we're missing a description of how the Quarg is speaking. Is there there a certain tone in its voice that we can pick up on? Are there any discernable facial expressions that it's making as it speaks? Is it showing emotion through its scales like other Quarg? (Can it not as you mentioned on stream, and that be something the player character picks up on?) Something to break up the lore dump a little.